internal/examples/streaming/stream.pb.go

```diff
@@ -43,7 +46,7 @@ var _ = math.Inf
 // is compatible with the proto package it is being compiled against.
 // A compilation error at this line likely means your copy of the
 // proto package needs to be updated.
-const _ = proto.GoGoProtoPackageIsVersion2 // please upgrade the proto package
+const _ = proto.GoGoProtoPackageIsVersion3 // please upgrade the proto package
```

Comment:
All of our proto files, [including this one](https://github.com/yarpc/yarpc-go/blob/dev/internal/examples/protobuf/examplepb/example.proto) use the proto3 package